.github/workflows/gh-actions.yml

```diff
@@ -90,6 +90,8 @@ jobs:
 
   Conformance_Tests_ESNext_A:
     runs-on: ubuntu-20.04
+    env:
+      BUILD_OPTIONS: --buildoptions=--test262=ON
```

Comment:
run-tests --test262-esnext should implicitly trigger building with --test262=ON. We should be able to run test262 tests locally without any additional command line option.

Reply:
i see, thank you for the information !